_posts/2021-02-09-january-rubygems-update.md

```diff
@@ -0,0 +1,34 @@
+---
+title: January 2021 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in January.
+
+## RubyGems News
+
+In January, we released new versions of bundler `v2.2.7` and `v2.2.8`, and rubygems `v3.2.7` and `v3.2.8`, the most important change being to fix resolver edge cases and speed up bundler - [#4277](https://github.com/rubygems/rubygems/pull/4277). This change makes bundler faster and the resolver more correct. We also keep refining our RFC for Bundler Version Locking -[#29](https://github.com/rubygems/rfcs/pull/29) and plan to move on to the implementation soon.
```

Comment:
We also released bundler v2.2.5 and v2.2.6 in January (and corresponding versions of rubygems v3.2.5 and v3.2.6). We should mention all versions we released, and we can probablt say something like this to describe them in general instead of mentioning https://github.com/rubygems/rubygems/pull/4277 specifically:

> The main improvements in these releases are resolver correctness and better performance. Have a look a the changelogs for the specific changes: https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#328--2021-02-02 and https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#228-february-2-2021.

Reply:
alright, will update this part.

Reply:
The version numbers are different from what I added though is this in addition to the v2.2.7, v2.2.8 release?

Reply:
Yes, we released all 4 versions this month. You can check this in this page: https://rubygems.org/gems/bundler/versions (the rubygems versions are released the same day in sync, they're just 3.2.x, instead of 2.2.x).

Reply:
This is more accurate then.
> In January, we released new versions of Bundler `v2.2.5`, `v2.2.6`, `v2.2.7`, and `v2.2.8`, and corresponding versions of RubyGems `v3.2.5`, `v3.2.6`, `v3.2.7`, and `v3.2.8`. The main improvements in these releases are **resolver correctness** and **better performance**. Learn more about specific changes made from the changelogs: [Bundler Changelog](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#228-february-2-2021) and [RubyGems Changelog](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#328--2021-02-02).

Reply:
Yes, that looks perfect to me!

Reply:
great thank you!